RecoJets/JetPlusTracks/plugins/JetPlusTrackProducerAA.cc

```diff
@@ -158,13 +157,13 @@ void JetPlusTrackProducerAA::produce(edm::Event& iEvent, const edm::EventSetup&
     bool ok = false;
 
     if (!vectorial_) {
-      scaleJPT = mJPTalgo->correction(corrected, *oldjet, iEvent, iSetup, pions, muons, elecs, ok);
+      scaleJPT = mJPTalgo->correction(corrected, oldjet, iEvent, iSetup, pions, muons, elecs, ok);
       p4 = math::XYZTLorentzVector(corrected.px() * scaleJPT,
                                    corrected.py() * scaleJPT,
                                    corrected.pz() * scaleJPT,
                                    corrected.energy() * scaleJPT);
     } else {
-      scaleJPT = mJPTalgo->correction(corrected, *oldjet, iEvent, iSetup, p4, pions, muons, elecs, ok);
+      scaleJPT = mJPTalgo->correction(corrected, oldjet, iEvent, iSetup, p4, pions, muons, elecs, ok);
```

Comment:
Why do you have this line? It seems to me you never use `scaleJPT` outside this scope

Reply:
This is a method inside JetPlusTrackCorrector. scaleJPT can be used to estimate the size of corrections in pT. It is not used in producer but can be used with correction on fly. I would prefer to keep this method as it is.